.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## 🙌 Thanks For Making PR
+
+<!--
+Before submitting your PR, please make sure that all tests pass. 
+You can run all tests using `npm test`.
+-->
+
+#### Checklist
+<!--
+Example how to mark a checkbox :-
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the code style of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] My changes generate no new warnings.
+- [ ] I have run the test cases.
+- [ ] This PR is for Adding New Theme
+
+
```

Comment:
I think the checklist is a little too long. Also, I think it should be a comment again. A lot of people just leave these types of checklists as is.

I don't think we need to encourage people to describe their work. The titles and descriptions people choose are typically enough.

```suggestion
Before submitting your PR, please check the following:

- Your code follows the code style of this project (`npm run lint`).
- You have performed a self-review of your own code.
- You have run the test cases (`npm run test`).
-->
```

Reply:
hmm, I will change this.